### PR TITLE
fix: Table align right not work on title

### LINKED ISSUE #15846: In the Table component's column prop, setting the align property to right only aligns the table rows not the table header if sorting is enabled
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/7269lj9n76)

### Steps to reproduce
- Create an antd Table with the columns prop set to an array of objects
- In one of those objects set the align property to `right` and the sorter property to `(a, b) => a.age - b.age`
- Observe that the column header is aligned left and the rows are aligned right

### What is expected?
That the column header would also be aligned right

### What is actually happening?
That the column header would also be aligned left

| Environment | Info |
|---|---|
| antd | 3.16.1 |
| React | 16.8.6 |
| System | macOS 10.14.4 |
| Browser |  Google Chrome	73.0.3683.86 |

---
I believe this PR introduced the bug: #15758

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

fix #15846 

### 💡 Solution

Wrap an span for `text-align` work.

### 📝 Changelog

Fix Table align right not work on title.

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
@afc163 pls help to double confirm on this.

File: components/table/Table.tsx [hunk 1]
```diff
@@ -925,12 +925,14 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
...
         }),
         title: [
-          <div key="title" className={sortButton ? `${prefixCls}-column-sorters` : undefined}>
-            <span className={`${prefixCls}-column-title`}>
-              {this.renderColumnTitle(column.title)}
-            </span>
-            <span className={`${prefixCls}-column-sorter`}>{sortButton}</span>
-          </div>,
+          <span key="title" className={`${prefixCls}-header-column`}>
+            <div className={sortButton ? `${prefixCls}-column-sorters` : undefined}>
```
Comment (afc163) [inline components/table/Table.tsx]: `${prefixCls}-header-column` 和 `${prefixCls}-column-sorters` 能用同一个 node 么？感觉 dom 层级有点复杂了。

**Review by ztplz — APPROVED** [review]

File: components/table/Table.tsx (hunk 1 above)
Comment (zombieJ) [reply components/table/Table.tsx]: `display: table` 是块级的，要同一层就需要改成 `margin: 0 0 0 auto;` 来实现 `align: 'right'`。担心这样会有副作用，所以加了一层 span 让它延续上面的 `text-align: right`。

**Comment by ztplz** [comment]
@zombieJ  https://github.com/ant-design/ant-design/issues/15846#issuecomment-480466142

**Pull request merged**